compiler/src/iree/compiler/InputConversion/StableHLO/ConvertCollectives.cpp

```diff
@@ -236,6 +236,44 @@ static DenseIntElementsAttr convertToRankGroupsByCrossReplica(
   return DenseIntElementsAttr::get(type, newValues);
 }
 
+static DenseIntElementsAttr convertToRankGroupsByCrossPartition(
+    DenseIntElementsAttr partitionGroups, int32_t numReplicas,
+    OpBuilder &builder) {
+  if (numReplicas < 1) {
```

Comment:
I think I made the mistake, but we should use `<=` instead of `<`. Would you mind fixing all the places?